src/dg/dg.cpp

```diff
@@ -1916,6 +2039,10 @@ void DGBase<dim,real,MeshType>::allocate_system ()
     ghost_dofs.subtract_set(locally_owned_dofs);
     //dealii::DoFTools::extract_locally_relevant_dofs(dof_handler, locally_relevant_dofs);
 
+    // TO DO: QUESTION FOR PRANSHUL / DOUG:
+    /* Should all these aritifical dissipation variables be allocated here when artificial dissipation
```

Comment:
Hmm, yeah we probably don't need to allocate artificial dissipation if it's not being used. I will fix it in a future PR.